### DISCUSSION TITLE
Error when trying to run 'php artisan route:cache'

### DISCUSSION THREAD

**Discussion by PaulChase** [laravel/framework#38989] [Q&A]
- Laravel Version: 8..4
- PHP Version: 7.4.3
- Database Driver & Version: mysql  Ver 8.0.26-0ubuntu0.20.04.2 for Linux on x86_64 ((Ubuntu))

### Description: 
I have a live laravel website running on aws ebs.
I entered the admin panel with **laravel voyager** to run some php artisan commands with laravel voyager compass.
I first ran 'php artisan view:cache' and it cached sucessfully.
Then I tried 'php artisan route:cache' and 'php artisan config:cache' and there were giving me the error 
'Trying to access array offset on value of type null'
as also shown in the image below
 what can I do to solve this issue?

![Screenshot_20210918_130001](https://user-images.githubusercontent.com/81924071/133888640-2235227c-f36e-4354-8be8-d4c257a6ac3a.png)

**Comment by garygreen** [comment]
Should be asked on stackoverflow. But it looks like to me you haven't installed Laravel properly and it's missing config/.env.

**Comment by lk77** [comment]
i think your config/session.php is missing for some reason, or there is an issue with it, like a return null

try to use tinker and see what config('session') returns.

```
php -r "echo null['test'];"
PHP Notice:  Trying to access array offset on value of type null in Command line code on line 1
```

try to delete the cache manually,
remove bootstrap/cache/packages.php, bootstrap/cache/services.php
and storage/framework/cache/data/*
